x-pack/plugins/security_solution/public/flyout/document_details/right/components/status.tsx

```diff
@@ -77,7 +72,6 @@ export const DocumentStatus: FC = () => {
               contextId={scopeId}
               enrichedFieldInfo={statusData}
               scopeId={scopeId}
-              handleOnEventClosed={closeFlyout}
```

Comment:
why removing this handler?

Reply:
this is no longer necessary as while moving the I moved the `StatusPopoverButton` component within the `flyout` folder, I moved the `closeFlyout` directly in there (see [here](https://github.com/elastic/kibana/pull/190119/files/d0cf7eaecbcc7dff2890e5f24aba92de4679fb57#diff-aec56a19c4b730f1f6befd3246120cda48c6217c932cd69fcd85b309926fa393R49))